src/SleepForDays/SleepForDays.workflow.cs

```diff
@@ -0,0 +1,33 @@
+namespace TemporalioSamples.SleepForDays;
+
+using Microsoft.Extensions.Logging;
+using Temporalio.Workflows;
+
+[Workflow]
+public class SleepForDaysWorkflow
+{
+    private bool isComplete;
+
+    [WorkflowRun]
+    public async Task RunAsync()
+    {
+        isComplete = false;
```

Comment:
```suggestion
```

This is dangerous because it will overwrite a signal that comes before the workflow starts

Reply:
Good point, didn't consider this. Don't think this is a _real_ concern with this sample, but I get that this might not be a pattern we want to demonstrate to new users. Removed.